src/sgame/sg_bot.h

```diff
@@ -92,6 +92,8 @@ struct botMemory_t
 	int lastThink;
 	int stuckTime;
 	vec3_t stuckPosition;
+
+	int aliveTime;
```

Comment:
`spawnTime` would be more descriptive

Reply:
im not sure but i think alivetime matches better.
the time a bot is alive. like stucktime is the time that a bot is well stuck.
spawntime could maybe also be misunderstood as time it takes to spawn or similar.

Reply:
I agree with `aliveTime` being more explicit.

Reply:
aliveTime is good for the function which can be used in the behavior tree - amount of time the bot has been alive. I think spawnTime is better just for this variable, which contains the time the bot spawned.

Reply:
oh yes. that's absolute correct. 😲